src/modules/career/online-profiles.rst

```diff
@@ -236,9 +93,8 @@ handles, gamertags, or anything that is not a variation of your name.
 
 ``a.lovelace`` is a great GitHub profile name for Ada Lovelace!
 
-countessofcomputers is a poor GitHub profile name for Ada Lovelace –
-although she has a steeped background with computers, she should still
-choose a professional github profile name.
+countessofcomputers is a poor GitHub profile name for Ada Lovelace---although she has a steeped background with computers, she should still
```

Comment:
also countessofcomputers should have syntax highlighting if its the same kind of entity as a.lovelace